Week1/homework/app.js

```diff
@@ -1,11 +1,150 @@
 'use strict';
 
 {
+  // 1.1 declaring an array 
   const bookTitles = [
     // Replace with your own book titles
     'harry_potter_chamber_secrets',
+    "a_history_of_western_philosophy",
+    'sophie’s_world',
+    'meditations',
+    'zen_and_the_art_of_motorcycle_maintenance',
+    'man’s_search_for_meaning',
+    'the_essential-epicurus' ,
+    'meditations_on_first_philosophy',
+    'critique_of_pure_peason',
+    'beyond_good_and_evil'
   ];
-
+  //1.3 Make a function
   // Replace with your own code
-  console.log(bookTitles);
+  function myFavouriteBooks(books , ul){
+    for(let i = 0; i < books.length ; i++){
+      let listItem =  document.createElement('li');
+      listItem.innerText = books[i];
+      ul.appendChild(listItem);
+    }
+  }
+
+ //1.3 Make a function
+  function main(){
+  let newUl = document.createElement('ul');
+  newUl.style.listStyle = 'circle';
+  document.body.appendChild(newUl);
+  myFavouriteBooks(bookTitles ,newUl );
+ }
+  main();
 }
+
+//1.4 Make an object
+let objecBbookTitles = {
+  '11':{
```

Comment:
As I said on Slack it looks like the instructions on the id were not clear to you. What was meant is that you would create an object like this:
`const bookObject = {
	'harry_potter_chamber_secrets': {
		title: 'Harry Potter and the Chamber of Secrets',
		author: 'J.K. Rowling',
		language: 'English'
	}
}`

So you can access the book by it's id from the list like this: `bookObject[bookTitles[0]]`